loc/en.js

```diff
@@ -293,6 +293,7 @@ module.exports = {
       inputs: 'Inputs',
       ouputs: 'Outputs',
       sendCoins: 'Send coins',
+      timePending: 'Time pending',
```

Comment:
sounds strange, is it an official copy?

Reply:
yes, here https://projects.invisionapp.com/d/main?origin=v7#/console/20268544/426588440/preview?scrollOffset=16623